### PR TITLE
feat: add netlify rebuild periodic job

### PR DESCRIPTION
**Pull request by kushthedude**
Signed-off-by: Kush Trivedi <kushthedude@gmail.com>

### CODE REVIEW

File: .github/workflows/netlify-periodic-build.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: Trigger netlify build
+on:
+  schedule:
+  - cron: '30 3 * * *'
```
Comment (mrbobbytables) [inline .github/workflows/netlify-periodic-build.yml:4]:
```suggestion
  - cron: '0 */8 * * *'
```
I was thinking about this - I think we can get away with it firing to update every 8 hours. That should be 3 times a day and in general more than enough (at least for now XD). What do you think?

Comment (kushthedude) [reply .github/workflows/netlify-periodic-build.yml:4]: I agree but it depends upon the netlify plan as netlify started plan allows 300 build minutes/month and if I consider a build to last to take 3.5-4 minutes, that will almost exhaust 300 build minutes with just periodic deploys in a month.

Comment (mrbobbytables) [reply .github/workflows/netlify-periodic-build.yml:4]: Good point, we're on enterprise but I completely forgot that netlify build minutes cost $$$$.

We can stick with daily at 3:30am :)

**Pull request merged**